File: 01_gpio_digital/led_blink.py
import RPi.GPIO as GPIO
import time

LED_PIN = 4
GPIO.setmode(GPIO.BCM)              # GPIO.BCM or GPIO.BOARD
GPIO.setup(LED_PIN, GPIO.OUT)       # GPIO.OUT or GPIO.IN

for i in range(10):
    GPIO.output(LED_PIN, GPIO.HIGH)     # 1
    print("led on")
    time.sleep(1)
    GPIO.output(LED_PIN, GPIO.LOW)
    print("led off")
    time.sleep(1)

GPIO.cleanup()      # GPIO 핀 상태 초기화

File: 01_gpio_digital/led_blink.c
#include <wiringPi.h>
#define LED_PIN 4

int main (void) {
    //wiringPiSetup(); // WPi
    wiringPiSetupGpio(); // BCM
    pinMode (LED_PIN, OUTPUT);
    for (int i=0; i<5; i++) {
        digitalWrite (LED_PIN, HIGH); delay (700);      // HIGH=1
        digitalWrite (LED_PIN,  LOW); delay (300);      // LOW=0
    }
    return 0;
}
